monai/inferers/splitter.py

```diff
@@ -90,18 +113,18 @@ class SlidingWindowSplitter(Splitter):
     def __init__(
         self,
         patch_size: Sequence[int] | int,
-        patch_level: int = 0,
```

Comment:
May I ask why remove `patch_level` here, does it mean we can only process data in level 0?